addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1722,6 +1722,43 @@ QUnit.test('chat window should scroll to the newly posted message just after pos
     );
 });
 
+QUnit.test('chat window: post message on non-mailing channel with "CTRL-Enter" keyboard shortcut for small screen size', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_minimized: true,
+        mass_mailing: false,
+    });
+    await this.start({
+        env: {
+            device: {
+                isMobile: true,
```

Comment:
please add a comment that we don't actually want to target mobile device with this condition but small screen